Prospress/ruleset.xml

```diff
@@ -91,6 +91,7 @@
 		<properties>
 			<!-- e.g. body_class, the_content, the_excerpt -->
 			<property name="customAutoEscapedFunctions" type="array" value="woocommerce_wp_select"/>
+			<property name="customAutoEscapedFunctions" type="array" value="wcs_help_tip"/>
```

Comment:
@thenbrent I _think_ we might be able to just use one `customAutoEscapedFunctions` property with a comma separated list for the value e.g. `woocommerce_wp_select,wcs_help_tip`. I know I've seen/used this approach before but I can't find an example atm. Just trying to confirm